app/assets/javascripts/components/course_queue.js.jsx

```diff
@@ -9,6 +9,15 @@ var CourseQueue = React.createClass({
       instructorMessage: '',
     };
   },
+  updateTitle:function(){
+    var myRequest = this.getMyFirstRequest();
+    if (myRequest && this.state.requests.indexOf(myRequest.request) >= 0) {
+      var myRequestIdx = this.state.requests.indexOf(myRequest.request);
+      document.title = `(#${myRequestIdx + 1}) Office Hours Help Queue`
+    } else {
+      document.title = `(${this.state.requests.length}) Office Hours Help Queue`
```

Comment:
Thanks for putting this together! Thoughts on providing some indication as to whether or not we’re showing your request or the total queue size? Maybe `(8/24) Office Hours Help Queue` if you’re on the queue and `(24) Office Hours Help Queue` if not.

Reply:
Oh, 🤦‍♂ just realized you have a pound sign in there. That works for me!